src/libse/Forms/FixCommonErrors/FixDanishLetterI.cs

```diff
@@ -21,7 +21,7 @@ public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
                 var oldText = text;
 
                 // Make sure text contains lower: 'i'.
-                if (RegexUtils.LittleIRegex.IsMatch(text))
+                if (callbacks.AllowFix(p, Language.FixDanishLetterI) && RegexUtils.LittleIRegex.IsMatch(text))
```

Comment:
`RegexUtils.LittleIRegex.IsMatch(text)` is what prevented the execution bellow, it should've been only for `foreach (var regex in RegexUtils.DanishLetterI.DanishCompiledRegexList)`